pyvista/_vtk.py

```diff
@@ -451,3 +451,25 @@ def __init__(self):  # pragma: no cover
             """Raise version error on init."""
             from pyvista.core.errors import VTKVersionError
             raise VTKVersionError('vtkGLTFReader requires VTK v9 or newer')
+
+
+# lazy import as this was added in 9.1.0
+def lazy_vtkCameraOrientationWidget():
+    """Lazy import of the vtkCameraOrientationWidget."""
+    try:
+        from vtkmodules.vtkInteractionWidgets import vtkCameraOrientationWidget
+    except ImportError:  # pragma: no cover
+        raise ImportError('vtkCameraOrientationWidget requires vtk>=9.1.0')
+    return vtkCameraOrientationWidget()
+
+
+# Get the VTK version. The API here is consistent since at least VTK v4.0.0
+try:
+    _ver = vtkVersion()
+    vtk_version = (
+        _ver.GetVTKMajorVersion(),
+        _ver.GetVTKMinorVersion(),
+        _ver.GetVTKBuildVersion()
+    )
+except:
+    (4, 0, 0)
```

Comment:
My understanding is that in `pyvista.__init__` that it will raise an error if the version is less than 5.x.x.  If we are sure that this works for all versions 4.0.0+, then why do we need this try...except?

It is also probably fine to leave this as is in case there are some edge cases so that users can still proceed.  But I would add a warning here and maybe set the version to be `(0, 0, 0)` to make it really obvious.

Reply:
Sounds like thereás an assignment missing
```suggestion
    vtk_version = (4, 0, 0)
```